contracts/mocks/VotingMock.sol

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+import "../utils/Votes.sol";
+
+contract VotesMock is Votes {
+    constructor(string memory name) EIP712(name, "1") {}
+
+    function getTotalVotes() public view returns (uint256) {
+        return _getTotalVotes();
+    }
+
+    function getTotalAccountVotes(address account) public view returns (uint256) {
+        return _getTotalAccountVotes(account);
+    }
+
+    function getTotalVotesAt(uint256 timestamp) public view returns (uint256) {
```

Comment:
Why have a public view for something that is already public view ? Also, this is not a timestamp, its a block number